test/shared_setup.py

```diff
@@ -111,7 +111,7 @@ def test_rel_gitignore(self):
         try:
             self.setup.rel_gitignore(['testdata'], base_dir=base_dir)
         except Exception as e:
-            self.assertTrue('.pyc' in e.message, msg=e)
+            self.assertTrue('.pyc' in str(e))
```

Comment:
please rename `e` to `exc` or `err` while you're at it (we shouldn't be using single-letter variables outside of list comprehensions)